windows/src/developer/kmcmpdll/Compiler.cpp

```diff
@@ -1254,7 +1254,9 @@ DWORD ProcessSystemStore(PFILE_KEYBOARD fk, DWORD SystemID, PFILE_STORE sp)
 	case TSS_VERSION:
 		if((fk->dwFlags & KF_AUTOMATICVERSION) == 0) return CERR_VersionAlreadyIncluded;
 		p = sp->dpString;
-		if(*p < '5') AddWarning(CWARN_OldVersion);
+    if (wcstof(p, NULL) < 5.0) {
```

Comment:
Would it be possible to check against `VERSION_MIN`?

Reply:
VERSION_MIN is a hexadecimal 0x500, not a value 5.0. We could divide by 0x100 but it's less clear :)

Reply:
Note that the reason for this change is that version 10.0 was showing as a lower version than 5.0 with the old code.